exasol_integration_test_docker_environment/lib/base/base_task.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, *args, **kwargs):
         self.__hash = hash(self.task_id)
         self._init_non_pickle_attributes()
         self.register_required()
-        self._task_state = TaskState.NONE
+        self._task_state = TaskState.AFTER_INIT
```

Comment:
Do we really need that?

Reply:
the task states are useful for checking and debugging issue in the calling conventions of certain methods